flocker/provision/_libcloud.py

```diff
@@ -4,8 +4,10 @@
 Helpers for using libcloud.
 """
 
+from characteristic import attributes, Attribute
 
-def fixed_OpenStackNodeDriver_to_node(self, api_node):
+
+def _fixed_OpenStackNodeDriver_to_node(self, api_node):
```

Comment:
So is this function mostly copied from the libcloud project? Perhaps there should be a docstring with an attribution and a link to the issue that you've included below.